src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -404,9 +403,9 @@ private void SetTransferMode(TransferMode transferMode)
             }
         }
 
-        public async Task DisposeAsync() => await DisposeAsyncCore().ForceAsync();
+        public async Task DisposeAsync(CancellationToken cancellationToken = default) => await DisposeAsyncCore(cancellationToken).ForceAsync();
 
-        private async Task DisposeAsyncCore()
+        private async Task DisposeAsyncCore(CancellationToken cancellationToken)
```

Comment:
Not used inside the method -- so it it actually required?

Reply:
We want the cancellation token in dispose in case the receive task takes a while. @davidfowl wants to make the API change now but to hold off implementing the logic now because it might be a bit trickier than we think right now